DataFormats/Math/test/ExtVec_t.cpp

```diff
@@ -1,25 +1,21 @@
-#include "FWCore/Utilities/interface/GCC11Compatibility.h"
-#if GCC_PREREQUISITE(4,8,0)
-
 #include "DataFormats/Math/interface/ExtVec.h"
 
 #include<cmath>
 #include<vector>
 
 #include<iostream>
 
+#ifdef __AVX__
+#define CMS_USE_AVX
+#endif /* __AVX__ */
 
-// #include <x86intrin.h>
+// clang complains that it is not "standard"
+#define constexpr
```

Comment:
agree. this is limited to this test and for this PR
I wanted to avoid s/constexpr //g for the time being with the hope to get it compiling.
The main point of this PR is to get a version that compiles and run correctly with AVX2 so that others can continue working